x-pack/plugins/watcher/common/constants/watch_states.ts

```diff
@@ -6,26 +6,24 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const WATCH_STATES = {
-
+export const WATCH_STATES: { [key: string]: string } = {
   DISABLED: i18n.translate('xpack.watcher.constants.watchStates.disabledStateText', {
-    defaultMessage: 'Disabled'
+    defaultMessage: 'Disabled',
   }),
 
   OK: i18n.translate('xpack.watcher.constants.watchStates.okStateText', {
-    defaultMessage: 'OK'
+    defaultMessage: 'OK',
   }),
 
   FIRING: i18n.translate('xpack.watcher.constants.watchStates.firingStateText', {
-    defaultMessage: 'Firing'
+    defaultMessage: 'Firing',
   }),
 
   ERROR: i18n.translate('xpack.watcher.constants.watchStates.errorStateText', {
-    defaultMessage: 'Error!'
+    defaultMessage: 'Error!',
```

Comment:
nit: Is the exclamation point necessary here? Not sure what the guidelines are around context, just noticed we don't do the same for `Config error`

Reply:
inherited, but fixed